### PR TITLE
fix login issue

### LINKED ISSUE #4: [BUG?] Cannot login
```python
from steam.client import SteamClient


s = SteamClient()
s.verbose_debug = True
s.login("my_user", "my_password")

print("Logged on as: %s" % s.user.name)
print("Community profile: %s" % s.steam_id.community_url)
print("Last logon: %s" % s.user.last_logon)
print("Last logoff: %s" % s.user.last_logoff)
print("Number of friends: %d" % len(s.friends))

s.logout()
```

But `s.logged_on` is `False` and no info from account loaded in SteamClient. What am I doing wrong? Or it's a bug

### PR DESCRIPTION
**Pull request by detiam**
Resolve #4 
along some commits by others that fixed things that broken this project.

### CODE REVIEW

**Comment by StalkR** [comment]
Thanks @detiam, I was wondering why I was having 'login failed' errors today, and this PR fixes it. Steam must have enforced the new websocket today.

Looks good to me.

File: steam/core/cm.py [hunk 1]
```diff
@@ -509,7 +521,36 @@ def str_to_tuple(serveraddr):
...
+    def bootstrap_from_webapi_websocket(self):
+        """
+        Fetches CM server list from WebAPI and replaces the current one
+
+        :return: booststrap success
+        :rtype: :class:`bool`
+        """
+        self._LOG.debug("Attempting bootstrap via WebAPI for websocket")
 
+        from steam import webapi
```
Comment (WinterPhoenix) [inline steam/core/cm.py]: Why are you doing an import in a function?

**Review by WinterPhoenix — CHANGES_REQUESTED** [review]

File: steam/client/builtins/friends.py [hunk 2]
```diff
@@ -156,7 +156,7 @@ def add(self, steamid_or_accountname_or_email):
         """
         m = MsgProto(EMsg.ClientAddFriend)
 
-        if isinstance(steamid_or_accountname_or_email, (intBase, int)):
+        if isinstance(steamid_or_accountname_or_email, (int, int)):
```
Comment (StalkR) [inline steam/client/builtins/friends.py:159]: now that you removed `intBase`, you can replace `(int, int)` with just `int`

https://docs.python.org/3/library/functions.html#isinstance
> isinstance(object, classinfo)
> [...]
> If classinfo is a tuple of type objects (or recursively, other such tuples) or a [Union Type](https://docs.python.org/3/library/stdtypes.html#types-union) of multiple types, return True if object is an instance of any of the types

File: steam/core/cm.py (hunk 1 above)
Comment (detiam) [reply steam/core/cm.py]: It's from the original `bootstrap_from_webapi` function, do you want me to change it in this pr?
https://github.com/ValvePython/steam/commit/5f11d6d7cc15ba0eb810150bd5cd074fbab8840f#diff-1c8fa68928ae302603b5f8fa5bf7ca188dfcf186946269b33f367317bb5b2e68R443

File: steam/client/builtins/friends.py (hunk 2 above)
Comment (detiam) [reply steam/client/builtins/friends.py]: updated!

File: steam/core/cm.py (hunk 1 above)
Comment (WinterPhoenix) [reply steam/core/cm.py]: That's kinda weird...we'll leave it if it's something the original dev did. I'm not familiar enough with this codebase to categorically say it's without good reason.

**Pull request merged**